client/notification.js

```diff
@@ -291,7 +291,12 @@ _.extend(NotifyClient.prototype, {
    */
   setProxy: function(url) {
     this.apiClient.setProxy(url);
-  }
+  },
+
+  prepareUpload: function(pdf_data) {
+    if (pdf_data.length > 2 * 1024 * 1024) { throw "This file is above the 2MB limit."; }
```

Comment:
Do we need to add this error to the Error table in the documentation?

Reply:
do we do this check when sending precompiled? (if not, should we?)

Reply:
No we don't. We probably should, I'm not sure what that size limit is, unless we use the limit we set on the size for a request, which is 5MB

Reply:
Should I set size limit for precompiled to 5MB?